daprdocs/content/en/operations/troubleshooting/common_issues.md

```diff
@@ -226,3 +226,26 @@ In order for mDNS to function properly, ensure `Micorosft Content Filter` is ina
 Microsoft Content Filter is disabled when the output is "Success".
 
 > Some organizations will re-enable the filter from time to time. If you repeatedly encounter app-id values missing, first check to see if the filter has been re-enabled before doing more extensive troubleshooting.
+
+## admission webhook denied the request
+
+This is due to admission webhook has a whitelist for service accounts to create/modify resources.
```

Comment:
@willtsai here is the root cause, need add a link about how webhook work?

Reply:
@daixiang0 - I think I understand now. A link to learn webhook might be a good idea, but my suggestion is to be a little more descriptive on root cause, and what you are proposing to address the cause. Also, please replace the term "whitelist" with "allowlist". For example:
_"You may encounter an error similar to the one below due to admission webhook having an allowlist for service accounts to create or modify resources. You should <DESCRIBE ACTIONS THEY SHOULD TAKE> to address this error."_